### PR TITLE
Update app-governance-app-policies-create.md

### PR DESCRIPTION
**Pull request by nearora-pm**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@nearora-pm : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: CloudAppSecurityDocs/app-governance-app-policies-create.md [hunk 1]
```diff
@@ -16,11 +16,10 @@ Along with a built-in set of capabilities to detect anomalous app behavior and g
...
 You can create app policies from provided templates that can be customized, or you can create your own custom app policy.
 
-To create a new app policy, go to **Microsoft 365 Defender > App governance > Overview page > Policies**:
+To create a new app policy, go to **Microsoft 365 Defender > App governance > Overview page > Policies**. Select the **Create New Policy** option:
 
-- To create a new app policy with templates designed for app usage, select **Create policy** under **Create an app usage policy**.
-- To create a new app policy with templates designed for app permissions, select **Create policy** under **Create a permissions policy**.
-- To create a new app policy for app certification or for a custom policy, select **Create new**.
+- To create a new app policy with a template, choose the relevant template category followed by the template in that category. 
+- To create a custom policy, select the category Custom
```
Comment (dcurwin) [inline CloudAppSecurityDocs/app-governance-app-policies-create.md:22]:
```suggestion
- To create a custom policy, select the **Custom** category.
```

**Comment by PRMerger18** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/app-governance-app-policies-create.md [hunk 2]
```diff
@@ -16,11 +16,10 @@ Along with a built-in set of capabilities to detect anomalous app behavior and g
 
 You can create app policies from provided templates that can be customized, or you can create your own custom app policy.
 
-To create a new app policy, go to **Microsoft 365 Defender > App governance > Overview page > Policies**:
+To create a new app policy, go to **Microsoft 365 Defender > App governance > Overview page > Policies**. Select the **Create New Policy** option:
 
-- To create a new app policy with templates designed for app usage, select **Create policy** under **Create an app usage policy**.
-- To create a new app policy with templates designed for app permissions, select **Create policy** under **Create a permissions policy**.
-- To create a new app policy for app certification or for a custom policy, select **Create new**.
+- To create a new app policy with a template, choose the relevant template category followed by the template in that category.
```
Comment (dcurwin) [inline CloudAppSecurityDocs/app-governance-app-policies-create.md:21]:
```suggestion
- To create a new app policy from a template, choose the relevant template category followed by the template in that category. 
```

**Review by dcurwin — APPROVED** [review]

**Comment by PRMerger8** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**